### PR TITLE
Invalid crashpad handler behavior on x86 devices

### PR DESCRIPTION
**Pull request by konraddysput**
# Why

The crashpad handler path doesn't work correctly for x86 builds. It prevents the game from he startup because of the invalid library image. This diff adds:
* support for x86/x64 crashpad handler,
* different algorithm that determinates the crashpad path based on architecture path.

### CODE REVIEW

File: Runtime/Native/Windows/NativeClient.cs [hunk 1]
```diff
@@ -323,13 +323,20 @@ private string GetDefaultPathToCrashpadHandler()
...
-            string architecture = IntPtr.Size == 8 ? "x86_64" : "x86";
-
-            string pluginPath = Path.Combine(pluginDir, architecture);
-            string pluginHandlerPath = Path.Combine(pluginPath, crashpadHandlerName);
+            string[] availablePlugins = new string[2] { "x86_64", "x86" };
+            var pluginDirectoryPath = Path.Combine(Application.dataPath, pluginDir);
+            var pluginDirectory = new DirectoryInfo(pluginDirectoryPath);
+            if (!pluginDirectory.Exists)
+            {
+                return string.Empty;
```
Comment (perf2711) [inline Runtime/Native/Windows/NativeClient.cs:331]: small nit: wouldn't `null` be more appropriate here?

**Review by perf2711 — APPROVED** [review]
LGTM.

**Review by sarahedkins — APPROVED** [review]

**Review by rqbacktrace — APPROVED** [review]

**Review by vlussenburg — APPROVED** [review]

**Comment by konraddysput** [comment]
After first round of tests, new issues related to similar behavior was detected. I'm planning more changes.

File: Runtime/Native/Windows/NativeClient.cs (hunk 1 above)
Comment (konraddysput) [reply Runtime/Native/Windows/NativeClient.cs:331]: it's a better idea. thx

**Pull request merged**